plugins/github-deployments/README.md

```diff
@@ -0,0 +1,75 @@
+# GitHub Deployments Plugin
+
+The GitHub Deployments Plugin displays recent deployments from GitHub.
+
+![github-deployments-card](./docs/github-deployments-card.png)
+
+## Getting Started
+
+1. Install the GitHub Deployments Plugin
+
+```bash
+# packages/app
+
+yarn add @backstage/plugin-github-deployments
+```
+
+2. Add proxy and auth token for GitHub
+
+```yaml
+# app-config.yaml
+
+proxy:
+  ...
+  '/github/api':
+    target: https://api.github.com
+    changeOrigin: true
+    secure: true
+    headers:
+      Authorization:
+        # Content: 'token OAUTH-TOKEN'
+        $env: GITHUB_OAUTH_TOKEN
+```
+
+3. Add the plugin to the app
+
+```typescript
+// packages/app/src/plugins.ts
+
+export { plugin as GithubDeployments } from '@backstage/plugin-github-deployments';
+```
+
+4. Add the ... to the EntityPage:
+
+```typescript
+// packages/app/src/components/catalog/EntityPage.tsx
+
+import { EntityGithubDeploymentsCard } from '@backstage/plugin-github-deployments';
+
+const OverviewContent = ({ entity }: { entity: Entity }) => (
+  <Grid container spacing={3} alignItems="stretch">
+    // ...
+    <Grid item xs={12} sm={6} md={4}>
+      <EntityGithubDeploymentsCard entity={entity} />
```

Comment:
Think this can lose the entity pass now? or at least you can confirm with the code you have used to generate your screen shot.

Reply:
Good spot. I'll update this and rerun local testing for screenshot.